src/firefly/java/edu/caltech/ipac/firefly/server/query/UwsJobProcessor.java

```
@@ -161,12 +162,29 @@ HttpServiceInput createSingleResultInput(String jobUrl) {
     public String getJobUrl() { return jobUrl; }
 
     public DataGroup getResult(TableServerRequest request) throws DataAccessException {
-        HttpServiceInput input = createSingleResultInput(jobUrl).setFollowRedirect(true);            // ensure redirects are followed
+        HttpServiceInput input = createSingleResultInput(jobUrl).setFollowRedirect(false);            // no redirects
         try {
             //download file first: failing to parse gaia results with topcat SAX parser from url
             String filename = getFilename(jobUrl);
             File outFile = File.createTempFile(filename, ".vot", QueryUtil.getTempDir(request));
-            HttpServices.Status status = HttpServices.getData(input, outFile);
+
+            HttpServices.Status status= HttpServices.getData(input, (method -> {
+                    if(HttpServices.isOk(method)) {
+                        try {
+                            return HttpServices.defaultHandler(outFile).handleResponse(method);
+                        } catch (FileNotFoundException e) {
+                            return new HttpServices.Status(400,"Bad Request: Output file not found");
+                        }
```

You can ignore exception if you want since file was created above.